pkgs/tools/misc/topgrade/default.nix

```diff
@@ -46,6 +47,8 @@ rustPlatform.buildRustPackage rec {
     installManPage topgrade.8
   '';
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
I am pretty sure we got updates before without this line.

Reply:
With this, we can use `nix-shell maintainers/scripts/update.nix` to perform an update.

Reply:
We have mix-up date for that. It doesn't require any code changes, works out of the box and can do commits.